control-plane/plugin/src/resources/mod.rs

```diff
@@ -176,6 +176,27 @@ pub enum GetDrainArgs {
     Nodes,
 }
 
+/// Delete resources.
+#[derive(Debug, clap::Args)]
+pub struct DeleteArgs {
+    /// Ignore error if resource is not found.
+    #[clap(long, short, global = true)]
+    pub ignore_not_found: bool,
```

Comment:
Is this flag to ignore volume not present on mayastor and user executing the cmd to delete it? 

I see we check this flag if we get StatusCode::NOT_FOUND from mayastor delete_volume api

Reply:
It's to not exit process with error